warehouse/query-core/src/main/java/datawave/query/function/DocumentProjection.java

```diff
@@ -37,37 +37,42 @@ public class DocumentProjection implements DocumentPermutation {
      */
     private boolean trackSizes = true;
 
+    @Deprecated
     public DocumentProjection() {
         this(false, false);
     }
 
+    @Deprecated
     public DocumentProjection(boolean includeGroupingContext, boolean reducedResponse) {
-        this(includeGroupingContext, reducedResponse, true);
+        this.includeGroupingContext = includeGroupingContext;
+        this.reducedResponse = reducedResponse;
+        this.projection = new Projection();
     }
 
-    public DocumentProjection(boolean includeGroupingContext, boolean reducedResponse, boolean trackSizes) {
+    public DocumentProjection(boolean includeGroupingContext, boolean reducedResponse, boolean trackSizes, Set<String> projections,
+                    Projection.ProjectionType projectionType) {
         this.includeGroupingContext = includeGroupingContext;
         this.reducedResponse = reducedResponse;
-        this.projection = new Projection();
+        this.projection = new Projection(projections, projectionType);
         this.trackSizes = trackSizes;
     }
 
-    /**
-     * Set the delegate {@link Projection} with the fields to include
-     *
-     * @param includes
-     *            the set of fields to include
-     */
+    @Deprecated
+    public DocumentProjection(Set<String> projections, Projection.ProjectionType projectionType) {
```

Comment:
Add documentation that this constructor will not include the grouping context and will not reduce the document.